src/platform/packages/shared/kbn-test/src/jest/run.ts

```diff
@@ -90,9 +90,10 @@ export async function runJest(configName = 'jest.config.js'): Promise<void> {
   // Resolve Jest configuration
   const baseConfig = await resolveJestConfig(parsedArguments, resolvedConfigPath);
 
-  // Set up Scout reporter if enabled
-  if (SCOUT_REPORTER_ENABLED && resolvedConfigPath) {
-    process.env.JEST_CONFIG_PATH = resolvedConfigPath;
+  // Scout reporter relies on the JEST_CONFIG_PATH environment variable to be set
+  // to build its complete output
+  if (SCOUT_REPORTER_ENABLED) {
```

Comment:
Let's make sure we correctly set the `JEST_CONFIG_PATH` (see  https://github.com/elastic/kibana/pull/237247).

Reply:
Thanks! `deedddd` is a fun sha. Things are looking good, I'm loving these PRs. Thanks! My hope is for the team to take a look at this ASAP, ideally on Monday (@dmlemeshko will also be back then).